src/library/commandLoader.ts

```diff
@@ -25,6 +25,7 @@ export default class CommandLoader {
   }
 
   private static removeTemplateFile(files: string[]) {
-    return files.filter(file => file !== './src/commands/_template.ts');
+    const commandTemplateFile = './src/commands/_template.ts';
```

Comment:
I flip flopped about throwing this into the config file. It was nice to have it centralized, but then I thought it shouldn't really change or be different per-environment, so I decided it can just live in here then. It also doesn't get reused as well so it can live here until that changes.